dev/com.ibm.ws.el.3.0_fat/fat/src/com/ibm/ws/el/fat/tests/EL30MiscTests.java

```diff
@@ -91,7 +91,7 @@ public void testEL30ServiceLookup() throws Exception {
     /**
      * Comments in the META-INF/services/javax.el.ExpressionFactory files should be ignored
      *
-     * The BZ 64097 patch was applied to both EL-3.0 and th EL-2.2 API.
+     * The BZ 64097 patch was applied to both EL-3.0 and the EL-2.2 API.
      *
      * This one tests the service look using the jsp-2.2 feature because it relies on the el 2.2 api. This test was placed here for two mains reasons:
```

Comment:
Oops. Could you also update the wording here: "tests the service look**up** using"